src/constants.ts

```diff
@@ -1,16 +1,43 @@
-import { Functions } from './types';
+import { Config } from './types';
 
-export const defaultFunctions: Functions = {
-  internal: { tags: { dev: false, notice: true, return: true, param: true } },
-  external: { tags: { dev: false, notice: true, return: true, param: true } },
-  public: { tags: { dev: false, notice: true, return: true, param: true } },
-  private: { tags: { dev: false, notice: true, return: true, param: true } },
-} as const;
-
-export const defaultTags = {
-  tags: {
-    dev: false,
-    notice: true,
-    param: true,
+export const defaultConfig: Readonly<Config> = {
+  include: './**/*.sol',
+  exclude: '',
+  root: './',
+  functions: {
+    internal: { tags: { dev: false, notice: true, return: true, param: true } },
+    external: { tags: { dev: false, notice: true, return: true, param: true } },
+    public: { tags: { dev: false, notice: true, return: true, param: true } },
+    private: { tags: { dev: false, notice: true, return: true, param: true } },
+  },
```

Comment:
Is there a way to create the default config from the schemas defined `src/config/schemas.ts`?

Reply:
Yes there is, i've implemented this way in order to have better visibility of the full default configuration. We can change it though.